ettienekorayyi/poc-code-sydney
# Implemented Error Handling mechanism

## Rationale

Created a fallback UI that gets rendered when an error occurs

## Advice for Reviewers & Testing Notes
To test the feature
- run npm start
- once the react app loads on the browser disconnect your internet connection
- Click the code.sydney logo on the header
- Once the page refreshes, the fallback ui should be rendered.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.
![image](https://user-images.githubusercontent.com/19362078/80276224-9a942e00-872a-11ea-8cb6-9a144e5ca7b4.png)


## Task Name
DEV-015 Error Handling

## Linting Checklist
